src/Sarif.Multitool/QueryCommand.cs

```diff
@@ -13,6 +13,8 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
 {
     internal class QueryCommand : CommandBase
     {
+        private const int TooManyResults = 2;
```

Comment:
>TooManyResults [](start = 26, length = 14)

same note, s/be all caps.

Reply:
Opened discussion, per my reply to your other comment.

---
In reply to: [316743793](https://github.com/microsoft/sarif-sdk/pull/1653#discussion_r316743793) [](ancestors = 316743793)